src/cmr/concepts/concept.js

```diff
@@ -29,7 +29,7 @@ export default class Concept {
   constructor(conceptType, headers = {}, requestInfo = {}, params = {}) {
     // Set properties for data available during instantiation
     this.conceptType = conceptType
-    this.headers = headers
+    this.headers = { ...headers }
```

Comment:
Please add a comment here noted the need to copy versus assign for future devs that might not know, because on the surface this is an unnecessary change.